Makefile

```diff
@@ -457,6 +457,7 @@ endif
 	export ECR_HOST=`aws ecr get-authorization-token --region us-east-1 --output text --query 'authorizationData[].proxyEndpoint'`; \
 	export DOCKER_USER=AWS; \
 	export DOCKER_PASSWORD=$$ECR_TOKEN; \
+	export ACCELERATE_VERSION=$('`git status | grep HEAD | cut -d ' ' -f4`')
```

Comment:
Let's use the `git describe` version here too.

Reply:
indeed, thanks! I will remove this line, it has no role here.